### PR TITLE
fix: [SRM-14349]: [SRM-14480]: fix for loading and overflowing issue and delete will be disabled in downtime list for a single scenario

### PR DESCRIPTION
**Pull request by karansaraswat19**
### Summary

Downtime list delete button to be disabled when activeOrPastInstancesCount > 0

#### Screenshots
Screenshot 2023-04-05 at 5 57 06 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss [hunk 1]
```diff
@@ -58,3 +50,11 @@
     margin-top: var(--spacing-small);
   }
 }
+
+.tooltip {
+  max-width: 310px !important;
+
+  p {
+    font-size: 13px !important;
```
[the remark this answers is not in the document]
Comment (karansaraswat19) [reply src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss:58]: I couldn't find 13 px in the harness-design-system but in the mocks, 13 px is mentioned

[the remark this answers is not in the document]
Comment (praneshg239) [reply src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss:58]: then use the nearest font size available, use `12px` or `14px` 

```css
  --font-size-xsmall: 10px; /* very small text */
  --font-size-small: 12px; /* H4, small text */
  --font-size-normal: 14px; /* H3, normal text */
  --font-size-medium: 18px; /* H2 */
  --font-size-large: 28px; /* formerly 29px; need to be an even number */
  --font-size-xlarge: 32px;
```

File: src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss [hunk 2]
```diff
@@ -18,17 +18,9 @@
     font-size: 13px !important;
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss:18]: always use font size from design system

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss (hunk 2 above)
Comment (karansaraswat19) [reply src/modules/85-cv/pages/slos/SLODowntimePage/components/DowntimeList/DowntimeList.module.scss:18]: the mock had 13 pxs, Thani asked me to go with 13 pxs

**Comment by karansaraswat19** [comment]
retrigger cypress-cd

**Review by deepesh-ui — APPROVED** [review]
Lets plan on adding cypress for dowtime as these cases will be hard to test in jest

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

**Comment by karansaraswat19** [comment]
retrigger cypress-cd

**Comment by karansaraswat19** [comment]
retrigger cypress-cd

**Pull request merged**